Hevelyn (a3b5 b1+) (#3)
BrunoDamacena/wca-regulations-translations: portuguese-brazilian/wca-guidelines.md

- A2d1+) [ADIÇÃO] Se a assinatura de embaralhamento for descoberta faltando antes de uma tentativa, o quebra-cabeça deve ser retornado imediatamente ao embaralhador para verificação e para a assinatura correspondente.
- A2d1++) [ADIÇÃO] Uma tentativa não deve ser desqualificada exclusivamente devido a uma assinatura de embaralhamento faltando. Se a assinatura de embaralhamento estiver faltando depois do início de uma tentativa, o juiz tem que permitir que a tentativa continue (se ela ainda estiver acontecendo) e o Delegado da WCA deve permitir que o resultado seja mantido. Entretanto, o Delegado da WCA ainda pode substituir a tentativa com uma tentativa adicional se ele tiver razão (que não a assinatura faltando) para suspeitar que o embaralhamento estava incorreto.
- A2d1+++) [LEMBRETE] Apesar de poder o Delegado da WCA permitir que várias tentativas individuais com assinatura de embaralhamento faltando contem, ele deve garantir que assinaturas de embaralhamento faltando não se tornem comuns.

> - A3c1+) [ESCLARECIMENTO] Para o Clock, o competidor pode mudar a posição dos pinos durante a inspeção (ver Regra 12i1).

Reviewer: ```suggestion
- A3c1+) [ESCLARECIMENTO] Para o Clock, o competidor pode mudar a posição dos pinos durante a inspeção (ver [Regra 12i1](regulations:regulation:12i1).
```

> - A3c3+) [ESCLARECIMENTO] O competidor pode zerar o timer antes/durante a fase de inspeção se o juiz acidentalmente não o fez. Entretanto, é responsabilidade do juiz assegurar-se de que o timer está zerado (ver [Regra A3b](regulations:regulation:A3b)). Se o competidor acidentalmente tentar iniciar a resolução com um timer que o juiz não zerou (isto é: o competidor faz a inspeção, posiciona as mãos no timer por um período de tempo razoável, assume que o timer Stackmat irá iniciar normalmente, e começa a operar o quebra-cabeça), a tentativa deve ser substituída por uma tentativa adicional, a critério do Delegado da WCA.

Reviewer: ```suggestion
- A3c3+) [ESCLARECIMENTO] O competidor pode zerar o timer antes/durante a fase de inspeção se o juiz acidentalmente não o fez. Entretanto, é responsabilidade do juiz assegurar-se de que o timer está zerado (ver [Regra A3b](regulations:regulation:A3b)). Se o competidor acidentalmente tentar iniciar a resolução com um timer que o juiz não zerou (isto é: o competidor faz a inspeção, posiciona as mãos no timer por um período de tempo razoável, assume que o timer Stackmat irá iniciar normalmente, e começa a operar o quebra-cabeça), é recomendável que a tentativa seja substituída por uma tentativa adicional, a critério do Delegado da WCA.
```

- A3c3++) [ESCLARECIMENTO] O competidor não pode "testar" ou "resetar" o timer durante a inspeção, uma vez que a inspeção acaba quando o competidor inicia a resolução (ver [Regra A4d](regulations:regulation:A4d)). Se o competidor para o timer depois que ele o tenha começado pela primeira vez, isso é considerado parar a resolução (ver [Regra A6](regulations:regulation:A6)), mesmo que ele o faça dentro de 15 segundos do início da inspeção.

> - A3c4+) [ESCLARECIMENTO] Se foi determinado o uso de um objeto fino, o juiz não pode remover o objeto ao invés do competidor.

Reviewer: ```suggestion
- A3c4+) [ESCLARECIMENTO] Se foi determinado o uso de um objeto fino, o juiz não deve remover o objeto ao invés do competidor.
```

- A3c4++) [ESCLARECIMENTO] O competidor pode remover o objeto durante a resolução, se não o tiver feito durante a inspeção (p. ex. esqueceu-se).
- A4d1+) [ESCLARECIMENTO] Se a inspeção levou exatos 15 segundos (isso é, 15.00), a penalidade de tempo (+2 segundos) deve ser aplicada.

> - A4d2+) [ESCLARECIMENTO] Se a inspeção levou exatos 17 segundos (isso é. 17.00), haverá desqualificação da tentativa (DNF).

Reviewer: haverá desqualificação vs a tentativa deve ser desqualificada

> - A4d2+) [ESCLARECIMENTO] Se a inspeção levou exatos 17 segundos (isso é. 17.00), haverá desqualificação da tentativa (DNF).

Reviewer: ```suggestion
- A4d2+) [ESCLARECIMENTO] Se a inspeção levou exatos 17 segundos (isso é. 17.00), a tentativa deve ser desqualificada (DNF).
```

- A5b+) [ESCLARECIMENTO] Enquanto estiver inspecionando ou resolvendo o quebra-cabeça, o competidor pode tocar o quebra-cabeça com qualquer parte do corpo. Exceção: 3x3x3 com Uma Mão (ver [Regra C1b](regulations:regulation:C1b)).
- A5b++) [ESCLARECIMENTO] Se uma parte do quebra-cabeça do competidor se solta (ver [Regra 5a](regulations:regulation:5a)), o competidor não deve pedir por assistência em recuperar ou localizar a parte e é recomendável que o juiz tente prevenir de que outros auxiliem. Se alguém pegar a parte solta para o competidor, o juiz ou o competidor podem pedir para que ele a coloque na estação de resolução e para que ele não auxilie mais.

> - A5b+++) [ESCLARECIMENTO] Não é recomendado que o delegado da WCA conceda ao competidor uma tentativa extra somente com base na possibilidade de alguém ter pego a parte solta so quebra-cabeça do competidor.

Reviewer: ```suggestion
- A5b+++) [ESCLARECIMENTO] Não é recomendável que o Delegado da WCA conceda ao competidor uma tentativa extra somente com base na possibilidade de alguém ter pego a parte solta do quebra-cabeça do competidor.
```

- A6b+) [EXPLICAÇÃO] O valor arbitrário de 0,06 foi escolhido para levar em conta imprecisões devidas ao mau funcionamento do timer Stackmat.
- A6b1+) [EXPLICAÇÃO] O valor arbitrário de 0,06 foi escolhido para levar em conta imprecisões devidas ao mau funcionamento do timer Stackmat.
- A6e+) [ESCLARECIMENTO] Se o quebra-cabeça cair no colo do competidor após este ter parado o timer, isso é considerado como tocar o quebra-cabeça.

> - A6e++) [ESCLARECIMENTO] Se a Regra A6c for usada para penalizar uma tentativa, é recomebdado que as penalidades da Regra A6e sejam  aplicadas apenas se o competidor soltar o quebra-cabeça e tocá-lo novamente antes que o juiz o inspecione.

Reviewer: ```suggestion
- A6e++) [ESCLARECIMENTO] Se a [Regra A6c](regulations:regulation:A6c) for usada para penalizar uma tentativa, é recomendável que as penalidades da [Regra A6e](regulations:regulation:A6e) sejam aplicadas apenas se o competidor soltar o quebra-cabeça e tocá-lo novamente antes que o juiz o inspecione.
```

> - A7c+) [ADIÇÃO] Se um resultado assinado por um competidor estiver incompleto ou ilegível, deve ser interpretado da pior forma possível (p. ex. "1:05" será considerado 1:05.99, "25.X3" será considerado 25.73 se X puder ser um 1 ou um 7). Considerando que o timer Stackmat mostra um tempo de X:0Y.ss como X:Y.ss, uma dezena faltando deve ser considerada mais provável do que um dígito faltando (p. ex. "1:2.27" será considerado 1:02.27). O juiz que assinou um resultado ambíguo pode ser solicitado a esclarecê-lo, a critério do anotador de resultados.

Reviewer: ```suggestion
- A7c+) [ADIÇÃO] Se um resultado assinado por um competidor estiver incompleto ou ilegível, é recomendável que seja interpretado da pior forma possível (p. ex. "1:05" será considerado 1:05.99, "25.X3" será considerado 25.73 se X puder ser um 1 ou um 7). Considerando que o timer Stackmat mostra um tempo de X:0Y.ss como X:Y.ss, é recomendável que uma dezena faltando seja considerada mais provável do que um dígito faltando (p. ex. "1:2.27" será considerado 1:02.27). O juiz que assinou um resultado ambíguo pode ser solicitado a esclarecê-lo, a critério do anotador de resultados.
```

- A7c6+) [EXEMPLO] Exemplos de assinaturas que identificam o competidor: Nome, iniciais, ID da WCA, ID do participante na competição, um apelido sensato/aceitável.

> - A7c6++) [EXPLICAÇÃO] O ID do participante na competição é um identificador (p.ex. um número) atribuído a um competidor para a competição. Em algumas competições ele é utilizado para para manter o controle de planilhas de pontuação e inserir resultados.

Reviewer: ```suggestion
- A7c6++) [EXPLICAÇÃO] O ID do participante na competição é um identificador (p.ex. um número) atribuído a um competidor para a competição. Em algumas competições ele é utilizado para manter o controle de planilhas de pontuação e inserir resultados.
```

> - A7c6+++) [RECOMENDAÇÃO] A equipe de organização pode solicitar que os competidores assinem com um método específico de identificação (p. ex. O ID do participante na competição), mas não é recomendado que os competidores sejam desqualificados por usar um jeito diferente ao assinar.

Reviewer: ```suggestion
- A7c6+++) [RECOMENDAÇÃO] A equipe de organização pode solicitar que os competidores assinem com um método específico de identificação (p. ex. O ID do participante na competição), mas não é recomendável que os competidores sejam desqualificados por usar um jeito diferente ao assinar.
```

> - A7c6++++) [ADIÇÃO] É recomendado que competidores usem a mesma assinatura quando estiverem como juízes e/ou como embaralhadores durante a competição.

Reviewer: ```suggestion
- A7c6++++) [ADIÇÃO] É recomendável que competidores usem a mesma assinatura quando estiverem como juízes e/ou como embaralhadores durante a competição.
```

- A7g+) [EXEMPLO] Exemplos de incidentes que podem dar a um competidor uma tentativa adicional devido a inexperiência: não esperar a luz verde do timer, iniciar ou parar o timer incorretamente, exceder o período de inspeção em virtude de não entender como iniciar a tentativa, ou outras penas por erros procedimentais.
- A7g++) [ESCLARECIMENTO] Punições individuais não podem ser dispensadas. Uma tentativa adicional deve ser concedida.

> - A7g+++) [RECOMENDAÇÃO] O Delegado da WCA deve ter especial cuidado quando conceder mais de uma tentativa adicional para o mesmo competidor. Se mais de uma tentativa adicional for concedida, deve ser devido a um tipo de incidente diferente. Exemplo: se o Delegado da WCA já tiver dado uma tentativa adicional para um competidor por não ter esperado o bastante pela luz verde, este não deve dar uma outra tentativa adicional se o competidor não tiver esperado o bastante pela luz verde outra vez.

Reviewer: ```suggestion
- A7g+++) [RECOMENDAÇÃO] É recomendável que o Delegado da WCA tenha especial cuidado quando conceder mais de uma tentativa adicional para o mesmo competidor. Se mais de uma tentativa adicional for concedida, é recomendável que seja devido a um tipo de incidente diferente. Exemplo: se o Delegado da WCA já tiver dado uma tentativa adicional para um competidor por não ter esperado o bastante pela luz verde, é recomendável que este não dê uma outra tentativa adicional se o competidor não tiver esperado o bastante pela luz verde outra vez.
```

## <article-B><blindfolded><blindfoldedsolving> Artigo B: Resolução Vendada

> - B1+) [LEMBRETE] O competidor deve usar um quebra-cabeça sem texturas, marcas ou outras características que diferenciem peças similares (ver [Regra 3k](regulations:regulation:3k)). Deve ser dada especial atenção a esta regra para Resolução Vendada. Exceção: para o 3x3x3 Vendado, o quebra-cabeça pode ter logos permitidas por [Regra 3l](regulations:regulation:3l).

Reviewer: ```suggestion
- B1+) [LEMBRETE] O competidor deve usar um quebra-cabeça sem texturas, marcas ou outras características que diferenciem peças similares (ver [Regra 3k](regulations:regulation:3k)). É recomendável que especial atenção seja dada a esta regra para Resolução Vendada. Exceção: para o 3x3x3 Vendado, o quebra-cabeça pode ter logos permitidas por [Regra 3l](regulations:regulation:3l).
```
